test/unit/to-plain-object-test.js

```diff
@@ -0,0 +1,19 @@
+import test from 'blue-tape'
+import {toPlainObject} from "../../lib";
+
+test('toPlainObject', (t) => {
+    class TestClassObject  {
+        constructor(name) {
+            this.name = name;
+        }
+    }
+
+    const obj = new TestClassObject('class object');
+    const result = toPlainObject(obj);
+
+    t.equals(obj instanceof TestClassObject, true, 'obj is instanceof TestClassObject');
+    t.equals(obj, result, 'toPlainObject returns same object');
+    t.equals(typeof result.toPlainObject === 'function', true, 'enhanced object has function "toPlainObject"');
+    t.notEqual(result, result.toPlainObject(), 'obj.toPlainObject() returns copy');
```

Comment:
Should we also add a test to see what it actually is equal to?

Reply:
@TimBeyer  Good point, I added more tests.

Reply:
Actually the method doc string says 
> Mixes in a method to return just a plain object with no additional methods
But then it looks like it does copy everything including the methods.
Maybe we should check if it does what it promises, though I'm not sure how to proceed if it indeed does copy all the things.